### PR TITLE
initial version of diff command wrapper

### PR DESCRIPTION
**Pull request by davelopez**
Simple wrapper for the GNU diff command

### CODE REVIEW

File: tools/diff/macros.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<macros>
+    <token name="@TOOL_VERSION@">3.6</token>
```
Comment (bgruening) [inline tools/diff/macros.xml]: If this is just a few lines, you could move this macro directly into the main file.

File: tools/diff/diff.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<tool id="diff" name="diff" version="@TOOL_VERSION@+@WRAPPER_VERSION@">
+
+    <description>analyzes two files and generates a text file with information about the lines that are different.</description>
```
Comment (bgruening) [inline tools/diff/diff.xml]: Do you know if `diff` is part of the coreutils package? If so we should probably depend on this package.

File: tools/diff/macros.xml (hunk 1 above)
Comment (davelopez) [reply tools/diff/macros.xml]: Done!

File: tools/diff/diff.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
+<tool id="diff" name="diff" version="3.6+galaxy0">
+
+    <description>analyzes two files and generates a text file with information about the lines that are different.</description>
```
Comment (beatrizserrano) [inline tools/diff/diff.xml]:
```suggestion
    <description>analyzes two files and generates a text file with information about the lines that are different</description>
```

File: tools/diff/diff.xml (hunk 2 above)
Comment (davelopez) [reply tools/diff/diff.xml]: Added conda-forge package for diffutils 3.6 and requirement 😃

**Comment by bgruening** [comment]
Merci! Great contribution.

**Pull request merged**